### PR TITLE
Add 7.12.1 rules updates

### PR DESCRIPTION
**Pull request by brokensound77**
Add updates for 7.12.1 pre-built rules

Preview links: 
- [Prebuilt rule reference](https://security-docs_667.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-rules.html). 
- [Changelog](https://security-docs_667.docs-preview.app.elstc.co/guide/en/security/master/prebuilt-rules-changelog.html).

### CODE REVIEW

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc [hunk 1]
```diff
@@ -1,15 +1,17 @@
 [[prebuilt-rules-changelog]]
 == Prebuilt rule changes per release
 
-[float]
-=== 7.12.0
+beta[]
```
Comment (jmikell821) [inline docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:4]: @brokensound77 should we remove the `[beta]` warning? I thought we were out of beta, but can't remember for sure.

Comment (brokensound77) [reply docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:4]: should this change happen in 7.12.1 docs?

File: docs/detections/prebuilt-rules/rule-details/unusual-linux-network-connection-discovery.asciidoc [hunk 2]
```diff
@@ -105,6 +105,18 @@ Uncommon user command activity can be due to an engineer logging onto a server i
 
 *Framework*: MITRE ATT&CK^TM^
 
+* Tactic:
```
Comment (bm11100) [inline docs/detections/prebuilt-rules/rule-details/unusual-linux-network-connection-discovery.asciidoc:108]: If additional Mitre metadata is added, does that require a `Formatting only` version bump? There is no version bumps on the rules with Mitre changes.

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
 
-This prebuilt rule has been deprecated:
-
-* setgid-bit-set-via-chmod
+[float]
+=== 7.12.1
 
-These prebuilt rules have been updated:
+[float]
+=== 7.12.0
```
Comment (jmikell821) [inline docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:14]: Should we add a `N/A` here if there are no major updates? Looks a little odd with the standalone heading.

Comment (brokensound77) [reply docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:14]: oh, to `7.12.1` good catch. Or maybe `formatting changes only`

File: docs/detections/prebuilt-rules/rule-details/unusual-linux-network-connection-discovery.asciidoc (hunk 2 above)
Comment (brokensound77) [reply docs/detections/prebuilt-rules/rule-details/unusual-linux-network-connection-discovery.asciidoc:108]: these versions reflect the rule versions themselves. Mitre threat data will bump rule versions

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc [hunk 4]
```diff
@@ -1,15 +1,17 @@
...
 
+The following lists prebuilt rule updates per release. Only rules with
+significant modifications to their query or scope are listed. For detailed
+information about a rule's changes, see the rule's description page.
 
-This prebuilt rule has been deprecated:
-
-* setgid-bit-set-via-chmod
+[float]
+=== 7.12.1
```
Comment (brokensound77) [inline docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:11]:
```suggestion
=== 7.12.1

N/A
```

**Review by bm11100 — APPROVED** [review]

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc (hunk 1 above)
Comment (jmikell821) [reply docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:4]: @brokensound77 yes, this went GA in 7.11 so we can remove the `beta` tag.

Comment (brokensound77) [reply docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:4]: as opposed to 7.12.0

File: docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc [hunk 5]
```diff
@@ -1,15 +1,19 @@
 [[prebuilt-rules-changelog]]
 == Prebuilt rule changes per release
 
-[float]
-=== 7.12.0
+beta[]
```
Comment (brokensound77) [inline docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc:4]:
```suggestion
```

**Pull request merged**